src/plugin.js

```diff
@@ -102,54 +85,6 @@ class Cmcd extends Plugin {
 
 }
 
-function generateUuid() {
-  const uuid = new Uint8Array(16);
-
-  for (let i = 0; i < uuid.length; i++) {
-    uuid[i] = Math.floor(Math.random() * 256);
-  }
-
-  // Set the version number to 4
-  uuid[6] = (uuid[6] & 0x0f) | 0x40;
-
-  // Set the variant number to 2
-  uuid[8] = (uuid[8] & 0x3f) | 0x80;
-
-  // Convert the UUID to a string in the canonical format
-  const hexDigits = Array.from(uuid, byte => byte.toString(16).padStart(2, '0'));
-  const hexString = hexDigits.join('');
-
-  return `${hexString.slice(0, 8)}-${hexString.slice(8, 12)}-4${hexString.slice(13, 16)}-${String.fromCharCode((uuid[8] & 0x0f) | 0x80)}${hexString.slice(17, 20)}-${hexString.slice(20)}`;
-}
-
-function objToString(obj) {
-  const keys = Object.keys(obj);
-  const keyValPair = keys.map((key) => {
-    if (keyTypes[key] === 'string') {
-      return `${key}="${obj[key]}"`;
-    } else if (keyTypes[key] === 'boolean' && obj[key] === true) {
-      return `${key}`;
-    }
-    return `${key}=${obj[key]}`;
-
-  });
-
-  return keyValPair.filter(s => !s.includes('undefined')).join(',');
-}
-
-function toHeader(cmcdHeaders) {
```

Comment:
CML has a header generation utility that handles mapping the keys to the correct header shards.